<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MusicPlayer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/icon/res/logo.png</normaloff>:/icon/res/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="musicList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>441</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>781</width>
      <height>441</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>781</width>
      <height>441</height>
     </size>
    </property>
   </widget>
   <widget class="QSlider" name="positionSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>691</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>691</width>
      <height>21</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>691</width>
      <height>21</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">#positionSlider{
color:rgba(255, 255, 255, 0);
}</string>
    </property>
    <property name="maximum">
     <number>1</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="LabRatio">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>460</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>81</width>
      <height>21</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>510</y>
      <width>381</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>381</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>381</width>
      <height>25</height>
     </size>
    </property>
    <layout class="QHBoxLayout" name="volumeWidget">
     <item>
      <widget class="QToolButton" name="volumeBtn">
       <property name="minimumSize">
        <size>
         <width>26</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>26</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="icon.qrc">
         <normaloff>:/icon/res/MD_Volume_up.png</normaloff>:/icon/res/MD_Volume_up.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="volumeSlider">
       <property name="minimumSize">
        <size>
         <width>271</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>271</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="volumeLab">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QToolButton" name="playModeBtn">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>510</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/icon/res/UI_Loop.png</normaloff>:/icon/res/UI_Loop.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QCheckBox" name="listView">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>510</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/icon/res/UI_PlayerList.png</normaloff>:/icon/res/UI_PlayerList.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelInfo">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>311</width>
      <height>421</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>421</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelPicture">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>421</width>
      <height>421</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>421</width>
      <height>421</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>421</width>
      <height>421</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelMask">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>421</width>
      <height>421</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>421</width>
      <height>421</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>421</width>
      <height>421</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="icon.qrc">:/icon/res/Mask.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="next">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>500</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;下一个&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(239, 41, 41);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/icon/res/UI_Next.png</normaloff>:/icon/res/UI_Next.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="playpause">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>500</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/icon/res/UI_Play.png</normaloff>:/icon/res/UI_Play.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="previous">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>498</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;上一个&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/icon/res/UI_Previous.png</normaloff>:/icon/res/UI_Previous.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="addmusic"/>
    <addaction name="removemusic"/>
    <addaction name="cleanmusic"/>
   </widget>
   <widget class="QMenu" name="menumore">
    <property name="title">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/icon/res/menu.png</normaloff>:/icon/res/menu.png</iconset>
    </property>
    <addaction name="about"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menumore"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="addmusic">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/res/MD_Add.png</normaloff>:/icon/res/MD_Add.png</iconset>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </action>
  <action name="removemusic">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/res/MD_Remove.png</normaloff>:/icon/res/MD_Remove.png</iconset>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="cleanmusic">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/res/MD_Clear.png</normaloff>:/icon/res/MD_Clear.png</iconset>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </action>
  <action name="about">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/res/menu.png</normaloff>:/icon/res/menu.png</iconset>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
